oap-server/server-configuration/configuration-etcd/pom.xml

```diff
@@ -0,0 +1,178 @@
+                                <configuration>
+                                    <images>
+                                        <image>
+                                            <name>quayio/coreos-etcd:${etcd.version}</name>
+                                            <alias>etcd-client-integration-test</alias>
+                                            <run>
+                                                <ports>
+                                                    <port>etcd.port:2379</port>
+                                                </ports>
+                                                <wait>
+                                                    <time>5000</time>
+                                                </wait>
+                                                <entrypoint>
+                                                    <!-- exec form  -->
+                                                    <exec>
+                                                        <arg>/usr/local/bin/etcd</arg>
+                                                        <arg>--advertise-client-urls=http://0.0.0.0:2379</arg>
+                                                        <arg>--listen-client-urls=http://0.0.0.0:2379</arg>
+                                                    </exec>
+                                                </entrypoint>
+                                            </run>
+                                        </image>
+                                    </images>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>remove-it-etcd</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <plugin>
+                        <groupId>org.codehaus.gmaven</groupId>
+                        <artifactId>gmaven-plugin</artifactId>
+                        <version>1.5</version>
+                        <executions>
+                            <execution>
+                                <id>add-default-properties</id>
+                                <phase>initialize</phase>
+                                <goals>
+                                    <goal>execute</goal>
+                                </goals>
+                                <configuration>
+                                    <providerSelection>2.0</providerSelection>
+                                    <source>
+                                        project.properties.setProperty('etcd.host', 'localhost')
+                                        project.properties.setProperty('etcd.port', '2379')
```

Comment:
don't set `etc.port` here statically, which would cause conflict when multiple build jobs are running on the same Jenkins node, the port is dynamically determined by `docker-maven-plugin` (line 100), and is available as a system property(line 156-163)

Reply:
yes.